packages/status-react/package.json

```diff
@@ -46,7 +46,7 @@
     "@radix-ui/react-tooltip": "^0.1.7",
     "@radix-ui/react-visually-hidden": "^0.1.4",
     "@status-im/js": "0.1.0-alpha.2",
-    "@stitches/react": "^1.2.8",
+    "@stitches/react": "^1.2.6",
```

Comment:
todo: remove `styled-components` and `@types/styled-components after resolving all discussions